pkg/deployment/features/rebuild_out_synced_shards.go

```diff
@@ -0,0 +1,38 @@
+//
+// DISCLAIMER
+//
+// Copyright 2023 ArangoDB GmbH, Cologne, Germany
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Copyright holder is ArangoDB GmbH, Cologne, Germany
+//
+
+package features
+
+func init() {
+	registerFeature(rebuildOutSyncedShards)
+}
+
+var rebuildOutSyncedShards = &feature{
+	name:               "force-rebuild-out-synced-shards",
+	description:        "Force rebuild permanently out-synced shards due to a bug in ArangoDB 3.8-3.10",
+	version:            "3.8.0",
+	enterpriseRequired: true,
```

Comment:
Inconsistency: in README it says that this feature can be applied to community version too.